pkgs/development/tools/misc/texinfo/make-texinfo.nix

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
Name this file `common.nix` instead.

Reply:
[Previously @AndersonTorres requested](https://github.com/NixOS/nixpkgs/pull/289690#discussion_r1494454951) `common.nix` to be renamed to `make-texinfo.nix`. Is there some formalised guidance, e.g. in a README, on this?

Reply:
Not as far as I'm aware, but `common.nix` is much more common https://github.com/NixOS/nixpkgs/pull/289690#issuecomment-2149549761